packages/ruleset-migrator/src/utils/__tests__/isPackageImport.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { isPackageImport } from '../isPackageImport';
+
+describe('isPackageImport util', () => {
+  it.each([
+    'nimma/legacy',
+    'nimma',
+    'lodash',
+    'lodash/get',
+    'lodash/get.js',
+    '@stoplight/path',
+    '@stoplight/spectral-core',
+    '@stoplight/spectral-core/dist/file.js',
+  ])('given valid %s package import, should return true', input => {
+    expect(isPackageImport(input)).toBe(true);
+  });
+
+  it.each(['', '/nimma/legacy', 'path', 'https://cdn.skypack.dev/@stoplight/spectral-core'])(
```

Comment:
Is it worth also adding `spectral:oas` as an example here?

Reply:
could add it, though in reality `isPackageImport` is unlikely to ever receive `stoplight:oas` as it's being replaced with `@stoplight/spectral-rulesets` here https://github.com/stoplightio/spectral/blob/develop/packages/ruleset-migrator/src/transformers/extends.ts#L21